plugins/generator-1.15.2/forge-1.15.2/procedures/entity_heal.java.ftl

```diff
@@ -0,0 +1 @@
+((LivingEntity) ${input$entity}).setHealth(((${input$entity} instanceof LivingEntity)?((LivingEntity) ${input$entity}).getHealth():-1) + ${input$amount}f);
```

Comment:
You’re already casting in the beginning, so remove the extra instanceof in the middle and wrap the code with an if statement

Reply:
It's done. I changed some stuff but I was forced to cast because the `.setHealth()` and `.getHealth()` need a `LivingEntity` cast